WebAPI/CurrencyConverter/Controllers/CurrencyController.cs

```diff
@@ -57,5 +58,65 @@ public async Task<ActionResult<CurrencyDto>> GetExchangeRate(string baseName)
 
             return BadRequest("Unable to map models");
         }
+
+        [HttpDelete("{id}")]
+        public async Task<ActionResult> DeleteExchangeRate(int id)
```

Comment:
Should take name and delete all currencies with this name. Returns 204/202/200. Don't need mapping